src/manager/manager.go

```diff
@@ -268,6 +268,10 @@ func (manager *Manager) LoadNodeData() error {
 }
 
 func (manager *Manager) AddNode(node types.Node) error {
```

Comment:
Q:  all AddNode/RemoveNode requests are supposed to proxy to leader, is that right? Then why bother guard ?